.github/workflows/statictest.yml

```
@@ -1,43 +1,33 @@
+          latest: true
+          fileName: statictest
+          out-file-path: .tools
+          token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Setup autotest binary
+        run: |
+          chmod -R +x $GITHUB_WORKSPACE/.tools/statictest
+          mv $GITHUB_WORKSPACE/.tools/statictest /usr/local/bin/statictest
+
+      - name: Run statictest
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2046:warning:1:17: Quote this to prevent word splitting

(shellcheck)